src/context.py

```diff
@@ -0,0 +1,69 @@
+Features = {"CASEMAPPING": "rfc1459", "CHARSET": "utf-8", "STATUSMSG": {"@", "+"}, "CHANTYPES": {"#"}} # IRC server features (these are defaults)
```

Comment:
This comment doesn't need to exist (it was obvious just by looking at the line). The value for CASEMAPPING and CHARSET should both be enums